pins/boards.py

```diff
@@ -90,16 +90,15 @@ def pin_versions(self, name: str, as_df: bool = True) -> Sequence[VersionRaw]:
             Pin name.
 
         """
-        if not self.versioned:
-            raise NotImplementedError(
-                "Cannot show versions for a board type that does not support versioning."
-            )
 
         if not self.pin_exists(name):
             raise PinsError("Cannot check version, since pin %s does not exist" % name)
-
+        if isinstance(self, BoardRsConnect):
```

Comment:
related to #229,  Connect actually needs this to be True 😩

Reply:
Darn.

It might make it a bit less onerous to write

```python
detail = isinstance(self, BoardRsConnect)
```

instead of the assignment inside a block.